src/_tutorials/dart-vm/httpserver.md

```diff
@@ -527,24 +527,24 @@ Run the server and client on the command line.
 
 1. First the server:
 
-   {% prettify bash %}
+   ```terminal
    $ cd httpserver/bin
    $ dart basic_writer_server.dart
-   {% endprettify %}
+   ```
 
 2. Then, run the client in a new window:
 
-   {% prettify bash %}
+   ```terminal
    $ cd httpserver/bin
    $ dart basic_writer_client.dart
    Wrote data for Han Solo.
-   {% endprettify %}
+   ```
 
 3. The server writes the data to `file.txt`:
```

Comment:
We need to change this, too. How about:

Look at the data that the server writes to `file.txt`:

Reply:
Changed (with edits approved via chat).